nomad/plan_apply.go

```diff
@@ -395,6 +395,8 @@ func evaluateNodePlan(snap *state.StateSnapshot, plan *structs.Plan, nodeID stri
 		return false, "node is not ready for placements", nil
 	} else if node.Drain {
```

Comment:
i guess a node can be both frozen and drained? in that case both conditions should be outputted?

Reply:
yes, I'll add condition to cover this case